### DISCUSSION TITLE
A SVG-esque editor in pixijs?

### DISCUSSION THREAD

**Discussion by rayhomme** [pixijs/pixijs#8135] [Ideas]
Hi, 

Not a coder here. But a developer from a product manager pov. 

I'm looking to create a 2D SVG type editor that can work in conjunction with 3D Webgl. 

Think of a lightweight version of Illustrator or Inkspace. Janvas.com is good example of a SVG version of this in a browser. 

We need something that works like a CAD app with 2D and 3D together in the same space. Think Vectorworks. 

Here is a video of the tech challenge. 

https://www.youtube.com/watch?v=4MOhY4-LwjI&ab_channel=WilliamKnapp

The short term goal might be simpler. For example to have the 2D and 3D editors be completely separate. Just that the 2D and 3D can be viewed in the same space.

Any thoughts on the feasibility of doing this with pixijs?

**Comment by ivanpopelyshev** [comment]
Its possible.
